source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+==========
+Collations
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide shows you how to use **collations**, a set of sorting rules, to
+run operations using string ordering for specific languages and locales (a
+community or region that shares common language idioms).
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following French words: **cote**, **coté**, **côte**, and **côté**.
+
+The order in which they would appear using the default *binary
+collation* is:
+
+::
+
+   cote
+   coté
+   côte
+   côté
+
+If we apply the Canadian French collation, the order is:
+
+::
+
+   cote
+   côte
+   coté
+   côté
+
+.. note::
+
+   Collations are available in MongoDB 3.4 and later.
+
+-----
+Usage
+-----
+
+You can specify a collation when you create a **new collection** or
+**new index**. You can also specify a collation for :doc:`CRUD operations
+</fundamentals/crud>` and aggregations.
```

Comment:
Breaking the categories of collation usage into "new collection or new index" and "crud operations and aggregations" certainly makes sense, but I think it might be better to split them up differently. Something like:
```
You can specify a default collation for a collection when you create a new collection.
When creating an index, you can specify the collation used by that index. You can
also specify a collation when querying MongoDB using CRUD operations and
Aggregations.
```
I think capital "A" Aggregation is what we usually use in other docs (not sure how consistent we are about that in docs-node though).

Reply:
The server manual uses lowercase (unless it links to an article in title case or is at the beginning of a sentence) and `source/fundamentals/crud.txt` too. Which docs use capital "A"?

Reply:
I saw some recent changes to the Stitch docs that explicitly used a capital A -- pinging @nlarew for his perspective, since he can confirm if that's the norm.